resources/js/features/home/components/+root/HomeRoot.tsx

```diff
@@ -16,7 +17,9 @@ export const HomeRoot: FC = () => {
 
   return (
     <div className="flex flex-col gap-6">
-      {auth?.user ? null : <GuestWelcomeCta />}
+      {!auth?.user ? <GuestWelcomeCta /> : null}
+
+      {auth?.user?.isNew ? <NewUserCta /> : null}
```

Comment:
I misunderstood this. It looks like we're missing the "Newest Registered User" component too. Was that intentionally removed?

Reply:
It was intentionally removed because we gain a new user approximately every ~65s. I can add it back if you'd like; I suspect most users won't even notice it's missing.

Reply:
I didn't notice until trying to test this change. I can't imagine anyone would complain.